setup.py

```diff
@@ -7,4 +7,7 @@
                       'katpoint',
                       'matplotlib',
                       'numpy',
-                      'pyyaml'])
+                      'pyyaml'],
+    extras_require={
+        'live': ['katcorelib', 'katconf']
```

Comment:
This is a nice solution.
(Merging will likely create a conflict on PR #33, but we'll deal with that there)